src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditMoney.razor.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class EditMoney : ComponentBase
+{
+#nullable enable
```

Comment:
This one can be removed as we have nullable enabled in csproj

Reply:
Removed in latest commit